### PR TITLE
Updating placeholders in code and Deployment document

### PR DESCRIPTION
**Pull request by AkashVirani**
Fixes for issue [#103 - Remove excessive manual configurations for setting up Matter Center](https://github.com/Microsoft/mattercenter/issues/103)

### CODE REVIEW

File: tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.SharePointAppWeb/App_GlobalResources/Constants.resx [hunk 1]
```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="Central_Repository_Url" xml:space="preserve">
-    <value>[Enter URL of the central repository where all the common lists are stored, e.g. https://[TENANTNAME].sharepoint.com/sites/catalog]</value>
+    <value>https://MyTenant.sharepoint.com/sites/catalog</value>
```
Comment (premchandpl) [inline tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.SharePointAppWeb/App_GlobalResources/Constants.resx]: If we have the URL like this, there are chances that, user may think that URL has already been provided and he may not update the correct url. If we have placeholder like the old ones, the user will update the value

**Comment by amitabhm21** [comment]
We should think this through W.R.T the overall experience. Parking this for a round of discussion before we merge the changes.

**Comment by AkashVirani** [comment]
In deployment guide (Matter Center Build and Deployment Guide.docx), we have mentioned all the placeholders that user need to change as part of manual deployment (Appendix F) and deployment using Matter Center deployment script (Appendix H), so user will not be misguided in updating the URLs.
Also, with these changes user will need to modify 17 values in the solution instead of 69 values while deployment using Matter Center deployment script.

**Comment by WilsonReddy** [comment]
Adding  dummy values to the code instead of placeholders with actions and examples  confuses the users when they see the code. Its assumed to be the right values.  Keep placeholder test as is and make a new pull request.

**Pull request closed without merging**